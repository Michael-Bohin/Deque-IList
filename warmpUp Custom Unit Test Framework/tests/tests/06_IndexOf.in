new int 
add 190370559
add 645542413
add 21869
add 622067
add 950861745
add 622163906
add 6103464
add 681016848
add 704309464
add 363999672
count 
foreach
indexof 0
indexof 1
indexof 2
indexof 363999672
indexof 704309464
indexof 681016848
indexof 6103464
indexof 622163906
indexof 950861745
indexof 622067
indexof 21869
indexof 645542413
indexof 190370559
indexof 23654665
indexof 326841523
clear
add 38
indexof 38