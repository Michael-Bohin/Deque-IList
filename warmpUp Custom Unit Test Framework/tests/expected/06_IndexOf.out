10
190370559
645542413
21869
622067
950861745
622163906
6103464
681016848
704309464
363999672
-1
-1
-1
9
8
7
6
5
4
3
2
1
0
-1
-1
0